example-context/example-service/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java

```diff
@@ -44,7 +45,7 @@
     Enveloper enveloper;
 
     @Handles("cakeshop.add-recipe")
-    public void addRecipe(final JsonEnvelope command) throws EventStreamException {
+    public void addRecipe(final JsonEnvelope command) throws EventStreamException, InvalidSequenceIdException {
```

Comment:
Removed just a oversight snapshot service does not throw any more invalid sequence or duplicate snapshot ( as these scenarios cannot arise)